src/utils/runSortReferences.ts

```diff
@@ -182,11 +183,15 @@ const runSortReferences = () => {
           b: { title: string; time: number }
         ) => number
       ) => {
-        const pageTitle = getPageTitleValueByHtmlElement(sortContainer);
-        if (!pageTitle) {
+        const currentPageUID = getCurrentPageUid();
```

Comment:
The reason we can't use `getCurrentPageUid` is because this won't work for the sort icon in the sidebar. we'll need to derive the block uid from the sort container, returning the current page uid for the main one, or getting it from the rightSidebar methods if it's from the sidebar one.

Also, in general, we should use `window.roamAlphaAPI.ui.mainWindow.getOpenPageOrBlockUid()`, which is a promise the returns the current uid, instead of `getCurrentPageUid`

Reply:
> sort icon in the sidebar

Can you send me a screenshot of what you are referring to here?

Reply:
Nevermind, found it.

Reply:
Both `getOpenPageOrBlockUid()` and `getCurrentPageUid()` seem to work for the sidebar.


https://user-images.githubusercontent.com/3792666/216740126-bc080910-0512-47ca-9754-852061d71c11.mp4

Reply:
Right bc you are testing the same page in the sidebar as the one in the main window - test it with a _different_ page in the side bar